forte/data/data_store.py

```diff
@@ -26,9 +26,15 @@ def __init__(self):
         Each `entry data` in the `entry list` is represented by a list of attributes,
         For example, annotation type entry has the following format:
         [<begin>, <end>, <tid>, <entry_type>, <attr_1>, <attr_2>, ..., <attr_n>].
-
-        The first four fields are compulsory for every `entry data`. They are always in
-        the order of `begin`, `end`, `tid` and `entry type`, .
+        Group type entry has the following format:
+        [<member_type>, <[members_tid_list]>, <tid>, <entry_type>, <attr_1>,
+            <attr_2>, ..., <attr_n>].
+        Link type entry has the following format:
+        [<parent_tid>, <child_tid>, <tid>, <entry_type>, <attr_1>, <attr_2>, ..., <attr_n>].
+
+        The first four fields are compulsory for every `entry data`.
+        For example, first four fields of annotations entries are always in
+        the order of `begin`, `end`, `tid` and `entry type` .
         `Begin` and `end` are the begin and end character index of this entry in the text.
         `Entry type` is the type of this entry. `tid` is a unique id of every entry,
```

Comment:
Please be consistent with your documentation above, you wrote `entry_type` then we should keep the same.